### PR TITLE
docs: add deploy / dns / issue with the CLI

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: docs/verifiable-document/document-store.md [hunk 1]
```diff
@@ -6,19 +6,48 @@ sidebar_label: Deploying Document Store
...
-
-- Google Chrome web browser
-- Metamask setup, refer to the [Test Account Setup](/docs/appendix/ropsten-setup) section for the setup guide
-
 ## Deployment Methods
 
 Currently, we provide two ways of deploying the document store smart contract onto Ethereum:
 
-- [TradeTrust Admin Portal](https://admin.tradetrust.io/), a web app
+- [TradeTrust Admin Portal](https://admin.tradetrust.io/) or [Opencerts Admin Portal](https://admin.opencerts.io/) , a web app
```
Comment (gjj) [inline docs/verifiable-document/document-store.md:13]: formatting:
- i think Opencerts is stylized as OpenCerts
- additional whitespace after the link and before the comma

File: docs/verifiable-document/document-store.md [hunk 2]
```diff
@@ -6,19 +6,48 @@ sidebar_label: Deploying Document Store
...
 - [Open Attestation CLI](https://github.com/Open-Attestation/open-attestation-cli), a command line tool
 
-They are both decentralized apps (Dapps) as both enable you to interact with smart contracts on the Ethereum blockchain. In this guide however, we will only be using the TradeTrust Admin Portal since it provides a no-code option to deploy the document store.
+They are both decentralized apps (Dapps) as both enable you to interact with smart contracts on the Ethereum blockchain.
+
+## Using Open Attestation CLI
+
+### Prerequisites
+
+- [OpenAttestation CLI](../../component/open-attestation-cli) installed
```
Comment (gjj) [inline docs/verifiable-document/document-store.md:22]: link doesn't work (see https://deploy-preview-33--laughing-cori-52b566.netlify.app/docs/verifiable-document/document-store#prerequisites)

should change to `../component/open-attestation-cli` or `/docs/component/open-attestation/cli`

File: docs/verifiable-document/document-store.md [hunk 3]
```diff
@@ -6,19 +6,48 @@ sidebar_label: Deploying Document Store
...
+- A wallet on ropsten network (for instance using [Metamask](/docs/appendix/ropsten-setup))
+- The wallet private key
+- [Exposing the wallet private key to the CLI for write operations](../../component/open-attestation-cli#writing-operations)
```
Comment (gjj) [inline docs/verifiable-document/document-store.md:25]: link doesn't work here too, should remove the additional `../`

File: docs/verifiable-document/issuing-document.md [hunk 4]
```diff
@@ -8,12 +8,55 @@ After wrapping the documents and obtaining a merkle root, the documents are read
...
+- [TradeTrust Admin Portal](https://admin.tradetrust.io/) or [Opencerts Admin Portal](https://admin.opencerts.io/) , a web app
+- [Open Attestation CLI](https://github.com/Open-Attestation/open-attestation-cli), a command line tool
+
+They are both decentralized apps (Dapps) as both enable you to interact with smart contracts on the Ethereum blockchain.
+
+## Using Open Attestation CLI
+
+### Prerequisites
+
+- [OpenAttestation CLI](../../component/open-attestation-cli) installed
```
Comment (gjj) [inline docs/verifiable-document/issuing-document.md:24]: link doesn't work here too, should remove the additional ../

File: docs/verifiable-document/issuing-document.md [hunk 5]
```diff
@@ -8,12 +8,55 @@ After wrapping the documents and obtaining a merkle root, the documents are read
...
+- A wallet on ropsten network (for instance using [Metamask](/docs/appendix/ropsten-setup))
+- The wallet private key
+- [Exposing the wallet private key to the CLI for write operations](../../component/open-attestation-cli#writing-operations)
```
Comment (gjj) [inline docs/verifiable-document/issuing-document.md:27]: link doesn't work here too, should remove the additional `../`

File: docs/verifiable-document/wrapping-document.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ In this guide, we will learn how to generate the checksum by running the `wrappi
 
 ## Prerequisites
 
-- [OpenAttestation CLI](../component/open-attestation-cli) installed
+- [OpenAttestation CLI](../../component/open-attestation-cli) installed
```
Comment (gjj) [inline docs/verifiable-document/wrapping-document.md:15]: this too: https://deploy-preview-33--laughing-cori-52b566.netlify.app/docs/verifiable-document/wrapping-document#prerequisites

File: docs/verifiable-document/document-store.md (hunk 1 above)
Comment (Nebulis) [reply docs/verifiable-document/document-store.md:13]: done

File: docs/verifiable-document/document-store.md (hunk 2 above)
Comment (Nebulis) [reply docs/verifiable-document/document-store.md:22]: done

File: docs/verifiable-document/document-store.md (hunk 3 above)
Comment (Nebulis) [reply docs/verifiable-document/document-store.md:25]: done

File: docs/verifiable-document/issuing-document.md (hunk 4 above)
Comment (Nebulis) [reply docs/verifiable-document/issuing-document.md:24]: done

File: docs/verifiable-document/issuing-document.md (hunk 5 above)
Comment (Nebulis) [reply docs/verifiable-document/issuing-document.md:27]: done

File: docs/verifiable-document/wrapping-document.md (hunk 6 above)
Comment (Nebulis) [reply docs/verifiable-document/wrapping-document.md:15]: done

**Pull request merged**